Fixed FREQMAN "Range" files to use kHz vs KHz, added 2 HAM freq files, and use valid BW settings for WFM

NotherNgineer opened the pull request:
freqman_entry_get_step_from_str_short() function requires precise (case-sensitive) syntax for the step values read from frequency files.  See freqman_entry_steps_short table in freqman.cpp.

gullradriel commented:
What about using 200k when there should be an audible signal ? While 40k may be usefull for some situation, it's not good to hear when on a radio.
I don't know for HAM, so I trust your judgement.
After that, it can be merged as it's updating files to work better :-)

gullradriel commented:
Many thanks

NotherNgineer commented:
I agree about the default BW settings and I've updated them.  I have also changed the channel step for FM_STANDARD to 100kHz per ITU 1984 if you're OK with that.

gullradriel reviewed (approved):
Thanks for the update
